src/main/java/com/geronimo/model/Profile.java

```diff
@@ -0,0 +1,41 @@
+package com.geronimo.model;
+
+
+import lombok.*;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Entity
+@ToString(callSuper = true)
+@EntityListeners(AuditingEntityListener.class)
+public class Profile {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String status;
+
+    @Lob
+    private byte[] picture;
+
+    private Date dateOfBirth;
+
+    @Column(name = "date_created")
+    @CreatedDate
+    protected Date dateCreated;
+
+    @Column(name = "last_updated")
```

Comment:
I believe there's a hibernate naming strategy for columns, e.g. transformation of camel case into snake case. It may look a bit harder than just writing a @Column annotation with a name everywhere, but if you forget to put this annotation you'll get a column with inapropriate name. Here's a link for an example of how to setup NamingStrategy: https://www.petrikainulainen.net/programming/tips-and-tricks/implementing-a-custom-namingstrategy-with-hibernate/

Reply:
default strategy is camel case into snake case, right?